pages/glossary/programmatically-determined-context.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Programmatically determined context
+key: programmatically-determined-context
+---
+
+The programmatically determined context of a piece of information (_e.g._ an element) is all the additional information that has a [programmatically determinable](https://www.w3.org/TR/WCAG21/#dfn-programmatically-determinable) [relationship](https://www.w3.org/TR/WCAG21/#dfn-relationships) to it.
+
+It includes, but is not restricted to, `aria-label` attribute, element that is referred by `aria-labelledby` or `aria-describedby` attributes, ancestor `label` elements, `label` elements referring the current element with a `for` attribute, ancestor elements, table header associated with the cell containing the information, headings, text inside the same paragraph (`p` element), table cell, list item, …
```

Comment:
You could also add "sentence" to the list.

From the WCAG definition of programmatically determined link context:
In HTML, information that is programmatically determinable from a link in English includes text that is in the same paragraph, list, or table cell as the link or in a table header cell that is associated with the table cell that contains the link. 

Note: "Since screen readers interpret punctuation, they can also provide the context from the current sentence, when the focus is on a link in that sentence."

https://www.w3.org/TR/WCAG21/#dfn-programmatically-determined-link-context

Reply:
And what about the title-attribute?

Reply:
I've updated the definition of programmatically determined context in #555